app/index.js

```diff
@@ -46,115 +46,12 @@ var NemoGenerator = yeoman.generators.Base.extend({
-        var exists = fs.existsSync(jarPath);
-        if (!exists) {
-          return "Please make sure you've got a valid selenium standalone jar path specified! Please see https://github.com/paypal/nemo-docs/blob/master/driver-setup.md for more details.";
-        }
-        return true;
-      }
-    }, {
-      type: 'list',
-      name: 'customSpec',
-      choices: ['Yes', 'No'],
-      message: 'Would you like to add a custom spec for your application? It will test for presence of text on your landing page.'
-
-    }, {
-      type: 'input',
-      name: 'targetBaseUrl',
-      default: 'http://localhost:8000',
-      message: 'What is the URL of your application landing page (where your first test should start)?',
-      when: function (answers) {
-        return (answers.customSpec === "Yes");
-      }
-    }, {
-      type: 'input',
-      name: 'landingPageLocator',
-      default: '#wrapper h1',
-      message: 'What CSS selctor will select distinct text on your landing page?',
-      validate: function (landingPageSelector) {
-        if (landingPageSelector !== "") {
-          return true;
-        }
-        return "You need to add a CSS selector for your test to use.";
-      },
-      when: function (answers) {
-        return (answers.customSpec === "Yes");
-      }
-    }, {
-      type: 'input',
-      name: 'landingPageText',
-      default: 'Hello, index!',
-      message: 'What text should appear on your application\'s landing page within the locator provided above?',
-      validate: function (homePageText) {
-        if (homePageText !== "") {
-          return true;
-        }
-        return "You need to add some text for your first test to check.";
-      },
-      when: function (answers) {
-        return (answers.customSpec === "Yes");
-      }
-    }, {
-      type: 'input',
-      name: 'deployedUrl',
-      message: 'What is your deployed application landing page URL (if different from your already supplied URL)',
-      when: function (answers) {
-        return (answers.customSpec === "Yes");
-      }
-    }, {
-      type: 'list',
-      name: 'sauceSetup',
-      choices: ['Yes', 'No'],
-      message: 'Would you like to set up SauceLabs to test on mobile browsers?'
-    }, {
-      type: 'input',
-      name: 'sauceUser',
-      message: 'What is your SauceLabs username?',
-      validate: function (username) {
-        if (username !== "") {
-          return true;
-        }
-        return "You need to provide a username.";
-      },
-      when: function (answers) {
-        return (answers.sauceSetup === "Yes");
-      }
-    }, {
-      type: 'input',
-      name: 'sauceKey',
-      message: 'What is your SauceLabs access key?',
-      validate: function (key) {
-        if (key !== "") {
-          return true;
-        }
-        return "You need to provide an access key.";
-      },
-      when: function (answers) {
-        return (answers.sauceSetup === "Yes");
-      }
-    }];
```

Comment:
I agree. It adds value. But I wanted to reduce the amount of questions and complexity in the main generator in order to give users the basic scaffold quickly. I think we should add sauce labs support as a sub generator. And also an option for perfecto mobile. Possibly other mobile testing options as well.